src/instructlab/config/init.py

```diff
@@ -90,9 +91,14 @@ def init(
     # the parameter source from the parent of the parent command.
     else:
         param_source = ctx.parent.parent.get_parameter_source("config_file")
-
+    try:
+        overwrite = False
+        if interactive:
+            overwrite = check_if_configs_exist()
+    except click.exceptions.Exit as e:
+        ctx.exit(e.exit_code)
```

Comment:
`ctx.exit` prints the usage here when we exit, is this what we'd want in this case?

Reply:
same behavior as below

Reply:
Let's keep it as-is then to be consistent.